src/zenml/utils/dashboard_utils.py

```diff
@@ -81,3 +83,42 @@ def print_run_url(run_name: str, pipeline_id: Optional[UUID] = None) -> None:
             "Pipeline visualization can be seen in the ZenML Dashboard. "
             "Run `zenml up` to see your pipeline!"
         )
+
+
+def show_dashboard(url: str) -> None:
+    """Show the ZenML dashboard at the given URL.
+
+    In native environments, the dashboard is opened in the default browser.
+    In notebook environments, the dashboard is embedded in an iframe.
+
+    Args:
+        url: URL of the ZenML dashboard.
+    """
+    environment = get_environment()
+    if environment in (EnvironmentType.NOTEBOOK, EnvironmentType.COLAB):
+        from IPython.core.display import display
+        from IPython.display import IFrame
+
+        display(IFrame(src=url, width="100%", height=720))
+
+    elif environment == EnvironmentType.NATIVE:
+        if handle_bool_env_var(ENV_AUTO_OPEN_DASHBOARD, default=True):
+            try:
+                import webbrowser
+
+                webbrowser.open(url)
```

Comment:
This webbrowser.open doesnt work on WSL. I wanted to fix that for a while, but there  is no real solution except here: https://github.com/python/cpython/issues/89752#issuecomment-1254943311 . Maybe we can try this suggestion in a try block?

```python
import os
import shutil


## maybe add these where we detect the OS to add another WSL type
from platform import uname

# source: https://www.scivision.dev/python-detect-wsl/
def in_wsl() -> bool:
    return 'microsoft-standard' in uname().release
    
# probably wrap in a try catch
if in_wsl():
    webbrowser.get("wslview %s").open(url)
```

Reply:
Ok dashboard opening should work under WSL, could you give it a try and see if it works?

Reply:
Update: worked! 🎉